sdk/java/pulumi/src/main/java/com/pulumi/provider/internal/schema/PackageSpec.java

```diff
@@ -58,6 +66,10 @@ public String getName() {
         return name;
     }
 
+    public String getNamespace() {
+	return namespace;
```

Comment:
Thank you for cleaning up all whitespaces. We probably need a linter. But - nit: this line doesn't look right?

Reply:
In another view it's fine but here it's shown as
<img width="448" alt="image" src="https://github.com/user-attachments/assets/e521000d-f4a8-4da8-8652-f87835071341" />

Reply:
Whoops, that's tabs vs.spaces, sorry about that, should be fixed now.